runelite-client/src/main/java/net/runelite/client/plugins/microbot/tutorialisland/TutorialIslandScript.java

```diff
@@ -481,6 +510,7 @@ public void CookingGuide() {
             hideRoofs();
             sleep(600);
             Rs2Keyboard.keyPress(KeyEvent.VK_ESCAPE);
+            Rs2Walker.walkTo(new WorldPoint(Random.random(3092,3095), Random.random(3092,3095),0));
```

Comment:
We could have problems finding the survival guide gate, sometimes the script randomly runs towards the building and gets stuck. this solves it by moving closer to the gate.